charts/rstudio-workbench/README.md.gotmpl

```diff
@@ -0,0 +1,97 @@
+{{ template "chart.header" . }}
+{{ template "chart.description" . }}
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}{{ template "chart.appVersionBadge" . }}
+
+{{ template "rstudio.disclaimer" . }}
+
+{{ template "rstudio.install" . }}
+
+## Required Configuration
+
+This chart requires the following in order to function:
+
+* A license key, license file, or address of a running license server. See the `license` configuration below.
+* A Kubernetes [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that contains the home directory for users.
+  * If `homeStorage.create` is set, a PVC that relies on the default storage class will be used. Most Kubernetes environments do not have a default
+    storage class that you can use with `ReadWriteMany` access mode out-of-the-box. In this case, we recommend you disable `homeStorage.create` and 
+    create your own `PersistentVolume` and `PersistentVolumeClaim`, then mount them into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters.
+  * If you cannot use a `PersistentVolume` to properly mount your users' home directories, you'll need to manually mount your NFS server via 
```

Comment:
Is there supposed to be more to this sentence?

Reply:
Yes 🤣  thanks - will push the rest of the sentence.